bin/s2nd.c

```diff
@@ -349,6 +351,7 @@ struct conn_settings {
     int prefer_low_latency;
     int enable_mfl;
     int session_ticket;
+    int session_cache;
```

Comment:
these could all be single bits, right?:

```
unsigned <name>:1;
```

same for `insecure` and `use_corked_io` below.

that would be 10bits, as opposed to 10 * 64bits.

Reply:
Sounds reasonable. Updated